docs/knowledge_base/tutorials/chatbot/index.md

```diff
@@ -0,0 +1,288 @@
+# Creating a Chatbot Website using Taipy
```

Comment:
The page title should not be set as a `H1`. Please use the Title header for that.

Reply:
I did not find how to do a Title header. Where is it in other tutorials? Can you fix it for me?

Reply:
OK, Let me show you.

Reply:
Sure